xray/commands/audit/generic/auditmanager.go

```
@@ -21,8 +22,54 @@ import (
 	"github.com/jfrog/jfrog-client-go/xray/services"
 )
 
-// GenericAudit audits the project found in the current directory using Xray.
+// GenericAudit audits all the projects found in the given workingDirs
 func GenericAudit(
+	xrayGraphScanParams services.XrayGraphScanParams,
+	serverDetails *config.ServerDetails,
+	excludeTestDeps,
+	useWrapper,
+	insecureTls bool,
+	args []string,
+	progress ioUtils.ProgressMgr,
+	requirementsFile string,
+	ignoreConfigFile bool,
+	workingDirs []string,
+	technologies ...string) (results []services.ScanResponse, isMultipleRoot bool, err error) {
+	if len(workingDirs) == 0 {
+		return singleGenericAudit(xrayGraphScanParams, serverDetails, excludeTestDeps, useWrapper, insecureTls, args, progress, requirementsFile, ignoreConfigFile, technologies...)
+	}
+	projectDir, err := os.Getwd()
+	if errorutils.CheckError(err) != nil {
+		return
+	}
+	var errorList []string
+	defer os.Chdir(projectDir)
```

Let's not ignore the error returned by the `Chdir` function.